app/controllers/api/v1/categories_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Api::V1::CategoriesController < ApiController
+
+  def index
+    categories = JAPI::Trebek.categories(options = {count: 4, offset: rand(10000)})
```

Comment:
your `rand` is zero indexed, so make sure to add 1 to the end result so that you dont accidentally get 0 (low probability, but still)